pages/_app.tsx

```diff
@@ -67,7 +67,7 @@ const DIDAdminPanel = ({ Component, pageProps: { ...pageProps } }: AppProps) =>
         <meta name="author" content="Israel Antonio Rosales Laguan" />
       </Head>
       <ApolloProvider client={client}>
-        {pathname === '/ingresar' ? (
+        {pathname === '/ingresar' || pathname === '/info/[memberCode]' ? (
```

Comment:
@Eghinner please create a task to use this route in the APP to create the QR

Reply:
Ehmm, direct change
https://github.com/Azordev/did-app/blob/82a2c29f5e874f5ed713ac46e1d94be1242cf555/src/pages/User/components/UserHeader/UserHeader.vue#L49

Reply:
oh ok, perfect!